content/development/regex_assembly.md

```diff
@@ -313,14 +313,15 @@ Processor name: `include`
 ### Arguments
 
 - Include file name (required): The name of the file to include, without suffix
+- Suffix replacements (optional): Any number of space separated two-tuples, where the first entry is the suffix to match and the second entry is the replacement. To use, write `--` after the include file name. Tuples are space separated
```

Comment:
> Any number of space separated two-tuples… Tuples are space separated

Is the repetition intentional? Maybe it is an essential point and needs repeating, but I wasn't sure if it was deliberately repeated when I read it.

Reply:
Nope, thanks.